std_execution.bs

```diff
@@ -669,6 +669,58 @@ Sender consumers could be made pipeable, but we have chosen to not do so.
 However, since these are terminal nodes in a pipeline and nothing can be piped after them, we believe a pipe syntax may be confusing as well as unnecessary, as consumers cannot be chained.
 We believe sender consumers read better with function call syntax.
 
+## All awaitables are senders ## {#design-awaitables-are-senders}
+
+Since C++20 added coroutines to the standard, we expect that coroutines and awaitables will be how a great many will choose to express their asynchronous code. However, in this paper, we are proposing to add a suite of asynchronous algorithms that accept senders, not awaitables. One might wonder whether and how these algorithms will be accessible to those who choose coroutines instead of senders.
+
+In truth there will be no problem because all awaitable types automatically model the `typed_sender` concept. The adaptation is transparent and happens in the sender customization points, which are aware of awaitables.
+
```

Comment:
> "all awaitable types"

You need to be careful and precise what you mean when you say "awaitable" here.
I found there was much confusion about the subtle nuances in what we mean when we say "awaitable".

There can be awaitable types that are only awaitable within certain contexts.
e.g. many of the folly::coro awaitable types extract some information from the awaiting coroutine's promise via the coroutine_handle passed to await_suspend(). You can't just pass any old coroutine_handle to such an awaitable and expect it to work here.

In the [P1288](https://wg21.link/P1288) paper the `awaitable` concept required that `await_suspend()` was invocable with `coroutine_handle<void>` as a proxy for accepting any `coroutine_handle` and thus being awaitable within any natural coroutine-context (i.e. without `await_transform`).